src/styles/site.css

```diff
@@ -46,11 +46,18 @@
   @apply fixed top-6 lg:h-[315px] lg:w-[376px];
 }
 
+<<<<<<< HEAD
+@media (max-width: 1024px) {
+.page-home .home-hero {
+    padding: 2em;
+  }
+=======
 .page-who-we-are .testimonial-section a,
 .page-who-we-are .testimonial-section a strong {
   @apply text-white;
 }
 
 .page-thrift-store h3 {
   @apply text-gray-900 text-2xl lg:text-3xl font-extrabold leading-9;
+>>>>>>> main
 }
```

Comment:
@bfaulk04 just need to fix up conflicts then we are gtg

Reply:
idk how to do that, all looks as intended to me

Reply:
just remove the portions that aren't needed and re-commit. That would be the `<<<` line portions or `>>>>>` . The `HEAD`  portion shows what is on your branch. The `main` is what's on main. So if there are styles that you've changed that are conflicting with what's there you can remove the previous ones and keep yours or keep both if they don't conflict in general but remove the conflict guards and then re-commit.